rosdoc2/verbs/build/builders/sphinx_builder.py

```diff
@@ -157,10 +158,10 @@ def ensure_global(name, default):
             importlib.import_module(exec_depend)
         except ImportError:
             pkgs_to_mock.append(exec_depend)
-    # todo(YV): If users provide autodoc_mock_imports in their conf.py
-    # it will be overwritten by those in exec_depends.
-    # Consider appending to autodoc_mock_imports instead.
-    autodoc_mock_imports = pkgs_to_mock
+    
+    autodoc_mock_imports.extend(pkgs_to_mock)
+
+    print(f"[rosdoc2] autodoc mock imports: '{{autodoc_mock_imports}}'")
```

Comment:
It would be good to only print this if it is non-empty, as for the vast majority of users this is not used.